apps/richie/vars/all/main.yml

```diff
@@ -0,0 +1,28 @@
+# Application default configuration
+
+# -- route
+richie_host: "richie.{{ project_name}}.{{ domain_name }}"
+
+# -- nginx
+richie_nginx_image_name: "fundocker/openshift-nginx"
+richie_nginx_image_tag: "1.13"
+richie_nginx_port: 8061
+
+# -- elasticsearch
+richie_elasticsearch_image_name: "fundocker/openshift-elasticsearch"
+richie_elasticsearch_image_tag: "6.2.4"
```

Comment:
we moved to 6.3

Reply:
I didn't run into any issue when upgrading. Yes this can be postponed. It raises the question of how to best keep this in sync with what we do in the richie repository... :thinking:

Reply:
As long as we maintain deployable apps in Arnold's repository, we'll need to make a new release for Arnold each time we upgrade one of those apps.

Reply:
I was thinking of a unique place in richie's repository where the ES image is referenced and we get it from this place for:
- docker-compose
- arnold
- etc.

Reply:
This place can be the docker-compose file and Arnold _et al._ can stick to it. Do you have something more automated in mind?

Reply:
ok. I didn't think it would be possible/easy to retrieve the info from a docker-compose file.

Reply:
:thinking: I am not sure to understand what you have in mind, but we'll discuss this IRL before declaring a new issue.